src/views/htmlcontent/test/app.component.spec.ts

```diff
@@ -270,14 +277,101 @@ describe('AppComponent', function (): void {
             ele = fixture.nativeElement;
         });
 
+        it('should have started showing messages after the batch start command', () => {
+            let dataService = <MockDataService> fixture.componentRef.injector.get(DataService);
+            dataService.sendWSEvent(batchStart);
+            fixture.detectChanges();
+
+            let results = ele.querySelector('#results');
+            expect(results).toBeNull('results pane is visible');
+
+            // Messages should be visible
+            let messages = ele.querySelector('#messages');
+            expect(messages).not.toBeNull('messages pane is not visible');
+            expect(messages.className.indexOf('hidden')).toEqual(-1);
```

Comment:
Now that I think about it, yeah, we should be checking the style attributes instead of class names. I think in this case it works to check the class name, but that's only the case because we tend to explicitly hide elements.